packages/calypso-e2e/src/lib/pages/my-home-page.ts

```diff
@@ -43,4 +44,14 @@ export class MyHomePage {
 	async validateTaskHeadingMessage( message: string ): Promise< void > {
 		await this.page.waitForSelector( selectors.taskHeadingMessage( message ) );
 	}
+
+	/**
+	 * Given a partial or full string, verify that a site generated from
+	 * a vertical design uses the vertical name as the default site title.
+	 *
+	 * @param {string} vertical Name of the vertical used as the site title.
+	 */
+	async validateSiteTitle( vertical: string ): Promise< void > {
```

Comment:
**suggestion:non-blocking**: While in the context of this project the parameter name of `vertical` makes sense, this method can have uses outside of validating that selected vertical name matches the site name.

For instance, in another e2e spec someone may want to validate the site name after making a change from Settings > General. In that case, the param name of `vertical` would be a little confusing.

Reply:
Thanks for that catch, Edwin! You are totally right and I've made the change in my most recent commit.